libraries/lib-time-and-pitch/StaffPad/TimeAndPitch.cpp

```diff
@@ -70,14 +70,13 @@ struct TimeAndPitch::impl
 
   SamplesReal fft_timeseries;
   SamplesComplex spectrum;
-  SamplesReal mag;
+  SamplesReal norm;
```

Comment:
Hi,
What motivated this change of name? At first sight I don't like this. "Magnitude" was correct and usual in DSP.

Reply:
... is this actually a mathematical shortcut to avoid `abs` ? Would you care to explain ?

Reply:
`std::abs` uses `hypotf` which was a significant bottleneck on Windows. It was rewritten in a very straightforward way using SIMD. I haven't measured it on different compilers, so I can say for sure if this change is much needed.

Reply:
Did you mean you _can't_ say for sure ?

Reply:
Yep, sorry. I'm just about rebasing these commits on top of my branch, but judging by the time tests take, I don't expect any significant difference.